metadata_mapper/lambda_shepherd.py

```diff
@@ -88,6 +96,16 @@ def map_collection(payload, context):
                 InvocationType="Event",  # invoke asynchronously
                 Payload=json.dumps(payload).encode('utf-8')
             )
+    return {
+        'statusCode': 200,
+        'body': {
+            'collection_id': collection_id,
```

Comment:
If we did send those enrichment exceptions in the return value, they could trickle up to get added to this return value.

Reply:
Yeah that could be useful